docs/sync.js

```diff
@@ -252,6 +252,12 @@ class Sync {
      * }
      */
     static initiateClientReset(path) {}
+
+    /**
+     * Returns `true` if Realm still has a reference to any sync sessions regardless of their state.
+     * If `false` is returned it means that no sessions currently exist.
+     */
+    static hasExistingSessions() {}
```

Comment:
I know this was added to help us with tests. What's the use case for end users?

Reply:
Very little. It is potentially helpful in a Client Reset situation or elsewhere where you want to be 100% sure the Realm is fully closed.

Reply:
I would consider making it a private API then. We should limit the API surface to only truly useful features. We can always make it public later if we think it's valuable.

Reply:
Sure, we can do that.